web/src/locales/en.json

```diff
@@ -16,6 +16,7 @@
     "no-results": "No results found for this search 🌍"
   },
   "button": {
+    "bluesky-share": "Check out this cool app: {{baseUrl}}",
```

Comment:
```suggestion
    "bluesky-share": "Check real-time global electricity emissions data: {{baseUrl}}",
```

How about this?

Reply:
I just put something there cause I knew you'd come up with something better ;)